src/utils.py

```diff
@@ -271,7 +270,7 @@ def sendNotification(title: str, body: str, e: Exception = None) -> None:
         return
     for url in urls:
         apprise.add(url)
-    assert apprise.notify(title=str(title), body=str(body))
+    # assert apprise.notify(title=str(title), body=str(body)) # not work for telegram
```

Comment:
Do you have an example where this doesn't work? For now, let's leave this uncommented and file an issue if so (or comment on #208)